scripts/reports/score_report.Rmd

```diff
@@ -0,0 +1,92 @@
+---
+title: "`r `r params$disease` score report"
+author: Delphi Forecast Team
+date: "Rendered: `r format(Sys.time(), '%Y-%m-%d %H:%M:%S')`"
+output:
+  html_document:
+    code_folding: hide
+    toc: True
+    # self_contained: False
+    # lib_dir: libs
```

Comment:
I'm wondering what these are about. Seem to be scattered in a couple of notebooks. I guess this is header copypasta?

Reply:
yea it's copy pasta. self-contained will bundle external artifacts (like images) inside the html, not sure what lib_dir is about. i was vaguely trying to get some consistency of this header across our Rmds, but not too hard.